hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/action/commit/FlinkDeleteHelper.java

```diff
@@ -50,6 +50,7 @@
     BaseDeleteHelper<EmptyHoodieRecordPayload, List<HoodieRecord<EmptyHoodieRecordPayload>>, List<HoodieKey>, List<WriteStatus>, R> {
 
   private FlinkDeleteHelper() {
+    super(ignored -> -1);
```

Comment:
wouldn't this affect flink users who do not set delete parallelism? as `deduceShuffleParallelism()` not used here but default value changed from 200 -> 0